modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -203,47 +203,73 @@ Each provider has its own configuration map that can be passed to the `genai.vec
 | model
 | STRING
 | The name of the model you want to invoke. +
-Supported values: `"textembedding-gecko@001"`
+ +
+Supported values: +
+ +
+
+label:new[Introduced in 5.17]:
+
+* `"textembedding-gecko@001"`
+
+label:new[Introduced in 5.19]:
+
+* `"textembedding-gecko@002"`
+* `"textembedding-gecko@003"`
+* `"textembedding-gecko-multilingual@001"`
+
 | `"textembedding-gecko@001"`
 
 | region
 | STRING
 | GCP region where to send the API requests. +
-Supported values:
-`"us-west1"`,
-`"us-west2"`,
-`"us-west3"`,
-`"us-west4"`,
-`"us-central1"`,
-`"us-east1"`,
-`"us-east4"`,
-`"us-south1"`,
-`"northamerica-northeast1"`,
-`"northamerica-northeast2"`,
-`"southamerica-east1"`,
-`"southamerica-west1"`,
-`"europe-west2"`,
-`"europe-west1"`,
-`"europe-west4"`,
-`"europe-west6"`,
-`"europe-west3"`,
-`"europe-north1"`,
-`"europe-central2"`,
-`"europe-west8"`,
-`"europe-west9"`,
-`"europe-southwest1"`,
-`"asia-south1"`,
-`"asia-southeast1"`,
-`"asia-southeast2"`,
-`"asia-east2"`,
-`"asia-east1"`,
-`"asia-northeast1"`,
-`"asia-northeast2"`,
-`"australia-southeast1"`,
-`"australia-southeast2"`,
-`"asia-northeast3"`,
-`"me-west1"`
+ +
+Supported values: +
+ +
+
+* `"us-west1"`
+* `"us-west2"`
+* `"us-west3"`
+* `"us-west4"`
+* `"us-central1"`
+* `"us-east1"`
+* `"us-east4"`
+* `"us-south1"`
+* `"northamerica-northeast1"`
+* `"northamerica-northeast2"`
+* `"southamerica-east1"`
+* `"southamerica-west1"`
+* `"europe-west2"`
+* `"europe-west1"`
+* `"europe-west4"`
+* `"europe-west6"`
+* `"europe-west3"`
+* `"europe-north1"`
+* `"europe-central2"`
+* `"europe-west8"`
+* `"europe-west9"`
+* `"europe-southwest1"`
+* `"asia-south1"`
+* `"asia-southeast1"`
+* `"asia-southeast2"`
+* `"asia-east2"`
+* `"asia-east1"`
+* `"asia-northeast1"`
+* `"asia-northeast2"`
+* `"australia-southeast1"`
+* `"australia-southeast2"`
+* `"asia-northeast3"`
+* `"me-west1"`
 | `"us-central1"`
+
+| taskType
+| STRING
+| The intended downstream application (see link:https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings#api_changes_to_models_released_on_or_after_august_2023[provider documentation]). The specified `taskType` will apply to all resources in a batch. label:new[Introduced in 5.19]
+| label:optional[]
+
+| title
+| STRING
+| The title of the document that is being encoded (see link:https://cloud.google.com/vertex-ai/generative-ai/docs/embeddings/get-text-embeddings#api_changes_to_models_released_on_or_after_august_2023[provider documentation]). The specified `title` will apply to all resources in a batch. label:new[Introduced in 5.19]
+| label:optional[]
```

Comment:
Not sure about these "optional" labels, but maybe we can make them look more distinct from the "required" ones?

Reply:
I can add a card for that but we won't have time for it before release. I am more unsure about the location of "Required" and "Optional" labels in the "Default" column - seems misplaced? Would be better to have label in the "Key" column right?

Reply:
Before this PR we only had "required", as a way of saying that there _is_ no default for that setting. I still think that makes sense. I agree that "optional" label may not fit as well in that column. I think it could probably also just be left out.